src/mixins/UserConfig.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @copyright Copyright (c) 2020 John Molakvoæ <skjnldsv@protonmail.com>
+ *
+ * @author John Molakvoæ <skjnldsv@protonmail.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { emit, subscribe, unsubscribe } from '@nextcloud/event-bus'
+import { generateUrl } from '@nextcloud/router'
+import { loadState } from '@nextcloud/initial-state'
+import axios from '@nextcloud/axios'
+
+const eventName = 'photos:user-config-changed'
+
+export default {
+	data() {
+		const croppedLayoutLocalStorage = localStorage.getItem('photos:croppedLayout')
```

Comment:
```suggestion
		const croppedLayoutLocalStorage = localStorage.getItem('photos:cropped-layout')
```
Funny we had a similar discussion today on another repo :p
https://github.com/nextcloud/nextcloud-vue/pull/1457#discussion_r505550454

Reply:
Ah, it's because we use `localStorage.setItem('photos:' + setting, value)` later, right.
Let's keep it like that, it's no big deal. It's not like it is a `nextcloud:xxx` one :+1:

Reply:
I think it's cool to have the name of the app only, we know we're on nextcloud namespace anyway

Reply:
> I think it's cool to have the name of the app only, we know we're on nextcloud namespace anyway

No, that's not it. nextcloud: would not be fitting here anyway. This is for global events/variables only.
I was referring to the kebab/camel case :)